.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+python: '2.7'
+sudo: required
+services:
+  - docker
+  - postgresql
+addons:
+  apt:
+    update: true
+  postgresql: "9.2"
+  apt:
+    packages:
+      - postgresql-server-dev-9.2
+jobs:
+  include:
+  - stage: test
+    env: Type='virus scan'
+    script:
+      - sudo apt-get install clamav -y
+      - sudo freshclam
+      - clamscan --quiet -r ./
+  - stage: test
+    env: Type='unit tests'
+    install:
+      - cd chroma-manager
+      - pip install -r requirements.txt
+    before_script:
+      - psql -c "CREATE USER chroma;" -U postgres
+      - psql -c "ALTER USER chroma CREATEDB;" -U postgres
+      - psql -c "CREATE DATABASE chroma OWNER chroma;" -U postgres
+    script:
+      - export IML_DISABLE_THREADS=1
+      - python manage.py test tests/unit/
+  - stage: test
+    env: Type='behave tests'
+    install:
+      - cd chroma-manager
+      - pip install -r requirements.txt
+    before_script:
+      - psql -c "CREATE USER chroma;" -U postgres
+      - psql -c "ALTER USER chroma CREATEDB;" -U postgres
+      - psql -c "CREATE DATABASE chroma OWNER chroma;" -U postgres
+    script:
+      - export IML_DISABLE_THREADS=1
+      - behave --format plain tests/feature/cli/features
```

Comment:
Can we simplify all of the above into:
`make unit_tests` + `make behave_tests` + `make tests` (which just runs the previous two) so that people can easily, without having to know all of the required setup incantations, run tests on their local instance?

It will also simplify this `travis.yml`.

`make tests` is something that I think we should make consistent across all of our modules.  It's fine to decompose them into multiple `make *_tests` targets for finer grain testing, but `make tests` should be consistent across modules I think.

Reply:
maybe, I'll have to see how portable all the make stuff is to ubuntu.

Reply:
It should be very portable.

Reply:
Here is a run trying to use make:

https://travis-ci.com/intel-hpdd/intel-manager-for-lustre/jobs/126095048

I'm happy to revisit doing it this way later, but would rather focus on getting this out the door as is at the moment.